src/powerhouse/components/index.ts

```diff
@@ -1,7 +1,6 @@
 export * from './button';
 export * from './drag-and-drop';
 export * from './dropdown-menu';
-export * from './icon';
```

Comment:
temporarily removed these files in case the electron runtime still tried to import from the url despite the component not being used. this wasnt likely but still not something we want to be hassled with right now